### ISSUE TITLE
[Bug] Failing test: elephant/test/test_conversion.py::BinnedSpikeTrainTestCase::test_repr

### ISSUE THREAD

**Issue by sanjayankur31** [NeuralEnsemble/elephant#409]
**Describe the bug**
While building 0.10.0 on Fedora, we're seeing a test fail: `elephant/test/test_conversion.py::BinnedSpikeTrainTestCase::test_repr`

**To Reproduce**
1. Build elephant
2. Run `pytest` to run tests

We get this:

```
______________________ BinnedSpikeTrainTestCase.test_repr ______________________                                                                                             
                                                                                                                                                                             
self = <elephant.test.test_conversion.BinnedSpikeTrainTestCase testMethod=test_repr>                                                                                         
                                                                                                                                                                             
    def test_repr(self):                                                                                                                                                     
        train = neo.SpikeTrain(times=np.array([1.001, 1.002, 1.005]) * pq.s,                                                                                                 
                               t_start=1 * pq.s, t_stop=1.01 * pq.s)                                                                                                         
        bst = cv.BinnedSpikeTrain(train, t_start=1 * pq.s,                                                                                                                   
                                  t_stop=1.01 * pq.s,                                                                                                                        
                                  bin_size=1 * pq.ms)                                                                                                                        
>       self.assertEqual(repr(bst), "BinnedSpikeTrain(t_start=1.0 s, "                                                                                                       
                                    "t_stop=1.01 s, bin_size=0.001 s; "                                                                                                      
                                    "shape=(1, 10), format=csr_matrix)")                                                                                                     
E       AssertionError: 'Binn[19 chars]t=1.0, t_stop=1.01, bin_size=0.001; shape=(1, [19 chars]rix)' != 'Binn[19 chars]t=1.0 s, t_stop=1.01 s, bin_size=0.001 s; shap[25 char
s]rix)'                                                                                                                                                                      
E       - BinnedSpikeTrain(t_start=1.0, t_stop=1.01, bin_size=0.001; shape=(1, 10), format=csr_matrix)                                                                       
E       + BinnedSpikeTrain(t_start=1.0 s, t_stop=1.01 s, bin_size=0.001 s; shape=(1, 10), format=csr_matrix)
E       ?                             ++             ++                ++

elephant/test/test_conversion.py:684: AssertionError
```

**Expected behavior**
All tests should pass

**Environment**
 - OS (e.g., Linux): Fedora rawhide (development/35)
 - How you installed elephant (`conda`, `pip`, source): From source
 - Python version: platform linux -- Python 3.9.2, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
 - `neo` python package version: 0.9.0
 - `elephant` python package version: 0.10.0

 - (Any additional python package you want to include here)
The build and root logs are attached. These contain the full build output, and list all the software that is installed in the "buildroot".

[elephant-build-log.txt](https://github.com/NeuralEnsemble/elephant/files/6123620/elephant-build-log.txt)
[elephant-installed-packages.txt](https://github.com/NeuralEnsemble/elephant/files/6123621/elephant-installed-packages.txt)

**Comment by mdenker** [comment]
Thanks for the report!
I could reproduce the issue using both `pytest` and `nosetest` locally, will investigate what happened to the units.

**Comment by mdenker** [comment]
Fixed by #418
